index.html

```diff
@@ -47,9 +47,9 @@
         for title in response.css('h2.entry-title'):
             yield {'title': title.css('a ::text').extract_first()}
```

Comment:
I was reluctant on using `.get()` here because it requires latest parsel too. What do you think?